src/index.html

```diff
@@ -26,7 +26,11 @@
             :firmware-id="FC.CONFIG.flightControllerIdentifier"
             :hardware-id="FC.CONFIG.hardwareName"
         ></betaflight-logo>
-        <port-picker></port-picker>
+        <port-picker
+            v-model="PortHandler.portPicker"
+            :connected-devices="PortHandler.currentPorts"
+            :read-only="PortHandler.readOnly"
```

Comment:
This seems like it shouldn't work.

In the PortHandler I don't see `readOnly`, it seems to be called `portPickerReadOnly` there.

Also can't we just use `disabled`? This would unify it with the input fields.

Reply:
I will change and test it.

Reply:
Fixed.